scripts/docker-build

```
@@ -56,4 +56,15 @@ for file in "${assets[@]}"; do
   sha256sum "$file"
 done
 
+untracked_inputs() {
+  comm -1 -3 <(git ls-files | sort) <(sort out/build-inputs.txt)
+}
+if untracked_inputs | grep -q .; then
+  echo WARNING: BUILD IS NOT REPEATABLE
+  echo The following files were included in the build but are not tracked by git:
```

I'll admit that I find it very convenient, when developing, to include files in the build that are not necessarily tracked by git. Did this actually help with the reproducibility issue?

---

The reason that I chose to report untracked files rather than checking whether git is clean and that .dockerfile does not include anything not in git is precisely to give developers a bit of freedom.  The former is nice in that it fails fast but is probably too restrictive for developers.